strapi/strapi
# Take the playgroundAlways option into account for the graphql playground

### What does it do?

Make use of the `playgroundAlways` option in GraphQL configuration.

### Why is it needed?

Currently, the `playgroundAlways` option is ignored.

### How to test it?

/config/plugin.js
```javascript
{
  graphql: {
    enabled: true,
    
    config: {
      playgroundAlways: true
    }
  }
}
```

Run `NODE_ENV=production yarn develop`
Head to `localhost:1337/graphql`, the GraphQL playground should be available


### Related issue(s)/PR(s)

fix #11905
